<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportConfigDialog</class>
 <widget class="QDialog" name="ExportConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Important Items</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="exportToLabel">
       <property name="text">
        <string>Export to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="exportToLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="exportToButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="selectAllButton">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deselectAllButton">
       <property name="text">
        <string>Deselect All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exportButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ExportConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
